tools/recetox_msfinder/recetox_msfinder.xml

```diff
@@ -0,0 +1,76 @@
+<tool id="recetox_msfinder" name="RECETOX MsFinder" version="@TOOL_VERSION@+galaxy0">
+    <description>Annotation of fragment peaks in mass spectral libraries.</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator" />
+
+    <expand macro="requirements" />
+    <command>
+        <![CDATA[
+        sh ${write_param} &&
+        mono /MsFinder/MsFinder/bin/Debug/MsfinderConsoleApp.exe annotate -i "$input_data" -m 'MSFINDER.INI' -o "$annotated_data"        
+    ]]>
+    </command>
+
+    <configfiles>
+        <configfile name="write_param">
+            printf "%s\n" "Ms2Tolerance=$Ms2Tolerance" \
+            "IsTmsMeoxDerivative=$IsTmsMeoxDerivative" \
+            "RelativeAbundanceCutOff=$RelativeAbundanceCutOff" \
+            "Ms1Tolerance=$Ms1Tolerance" \
+			"MassToleranceType=$MassToleranceType" \
+			"StructurePredictionTimeOut=$StructurePredictionTimeOut" \
+			"Ocheck=$element_selection.Ocheck" \
+            "Ncheck=$element_selection.Ncheck" \
+            "Pcheck=$element_selection.Pcheck" \
+            "Scheck=$element_selection.Scheck" \
+            "Fcheck=$element_selection.Fcheck" \
+            "ClCheck=$element_selection.ClCheck" \
+            "BrCheck=$element_selection.BrCheck" \
+            "Icheck=$element_selection.Icheck" \
+            "SiCheck=$element_selection.SiCheck" \
+            "LewisAndSeniorCheck=$LewisAndSeniorCheck" \
+			"IsUseRtForFilteringCandidates=False" \
+			"RtToleranceForStructureElucidation=2.5" \
+			"IsUsePredictedRtForStructureElucidation=False" \
+			"Coeff_RtPrediction=-1" \
+			"Intercept_RtPrediction=-1" \
+			"IsUseRtInchikeyLibrary=True" \
+			"IsUsePredictedCcsForStructureElucidation=False" \
+			"IsUseCcsForFilteringCandidates=True" \
+			"CcsToleranceForStructureElucidation=10" \
+			"CanExcuteMS2AdductSearch=False" \
+            "IsUseXlogpPrediction=False" >MSFINDER.INI
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <expand macro="input" />
+        <expand macro="parameter" />
+        <section name="element_selection" title="Element Selection" expanded="true" help="Generate formula candidates that just contain the elements selected by the users.">
+            <expand macro="element_check_parameter" />
+        </section>
+    </inputs>
+
+    <outputs>
+        <expand macro="output" />
+    </outputs>
+
+    <tests>
+        <test>
+            <param name="input_data" value="test.msp" ftype="msp" />
+            <output name="annotated_data" file="out.msp" lines_diff="16" />
```

Comment:
What is the purpose of this test with such tolerance? Are we planning to improve that? Shouldn't that be reported somewhere?